basilisk/gui/history_msg_text_ctrl.py

```
@@ -506,21 +506,32 @@ def on_show_as_html(self, event: wx.Event | None):
-			self.speak_stream
+		return (
+			self.speak_response
 			and (
 				self.HasFocus()
 				or self.GetParent().prompt_panel.prompt.HasFocus()
 			)
 			and self.GetTopLevelParent().IsShown()
-		):
+		)
+
+	def append_stream_chunk(self, text: str):
```

Flake8 (7.2.0)

[warning] indentation contains tabs

(W191)